sql/sum.sql

```diff
@@ -93,8 +88,8 @@ create procedure sum_result (inout final any,
   --dbg_obj_print ('sorted = ', sorted);
   gvector_sort (sorted, 3, 2, 0);
   for (inx := 0; inx < length (sorted); inx := inx + 3)
-  tot	 := tot || cast (rdf_box_data (sorted[inx]) as varchar);
-  tot := __bft (tot, 2);
+    tot := tot || cast (rdf_box_data (sorted[inx]) as varchar);
+  -- tot := __bft (tot, 2); XXX: disabled until fixed excerpt to make ∞ loop
```

Comment:
Seems likely to be a typo ... or better to be reworded than left as is
```suggestion
  -- tot := __bft (tot, 2); XXX: disabled until fixed except to make ∞ loop
```